proxy/config.dist.php

```diff
@@ -91,6 +97,18 @@
   require "secrets.php";
 }
 
+// CiviCRM's API4 can authenticate with different flows
+// https://docs.civicrm.org/dev/en/latest/framework/authx/#flows
+// CiviProxy supports 'header', 'xheader', 'legacyrest', and 'param'.
+// api4_internal_auth_flow controls how CiviProxy sends credentials to CiviCRM, and
+// api4_external_auth_flow where CiviProxy looks for credentials on incoming requests.
+// The internal setting needs to have a single scalar value, but the
+// external setting can be an array of accepted flows.
+// There is no standard header for site key, so in both header and xheader
+// flows it uses X-Civi-Key
+$api4_internal_auth_flow = 'header';
+$api4_external_auth_flow = ['legacyrest'];
```

Comment:
Without having reviewed thoroughly, *AuthX* is not bound to APIv4, but allows authentication in general, also for *AuthX*'s APIv3 REST endpoint. I suppose your PR does not cover *AuthX* + APIv3 - that's basically ok, just wanted to note that …

Reply:
Yep, it would be great to get AuthX support for APIv3 as well, and shouldn't take too much more work. Let's file a separate issue for that and solve it in another PR.

Reply:
I suppose I should rename the settings to simply authx_internal_flow and authx_external_flow, without including the API version.